src/app/components/Uploader/index.styles.tsx

```diff
@@ -0,0 +1,53 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+
+const styles = {
+  container: ({ spacings, mq }: Theme) =>
+    css({
+      margin: `${spacings.TRIPLE}rem ${spacings.FULL}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        marginLeft: `${spacings.DOUBLE}rem`,
+        marginRight: `${spacings.DOUBLE}rem`,
+      },
+      [mq.GROUP_4_MIN_WIDTH]: { marginLeft: 0, marginRight: 0 },
+    }),
+  card: ({ spacings, palette }: Theme) =>
+    css({
+      backgroundColor: palette.WHITE,
+      paddingLeft: `${spacings.DOUBLE}rem`,
+      paddingRight: `${spacings.DOUBLE}rem`,
+    }),
+  heading: ({ spacings }: Theme) =>
+    css({
+      paddingTop: `${spacings.DOUBLE}rem`,
+      paddingBottom: `${pixelsToRem(12)}rem`,
+    }),
+  linkContainer: () =>
+    css({
+      display: 'flex',
+      justifyContent: 'start',
+    }),
+  linkBackground: ({ spacings, palette }: Theme) =>
+    css({
+      padding: `${pixelsToRem(11)}rem 
+      ${pixelsToRem(15)}rem 
+      ${pixelsToRem(13)}rem 
+      ${spacings.DOUBLE}rem`,
```

Comment:
Should the padding be equal for X and Y values, so top/bottom = ${pixelsToRem(11)}rem and left/right = ${pixelsToRem(15)}rem or something? Just looks like the button contents (text and icon) won't be exactly centered otherwise.

Reply:
Talked to UX and turns out they should be equal! So that makes life easier 😅